From d7e02c8991daf5f706d26303f7e9d7c742545083 Mon Sep 17 00:00:00 2001
From: Matus Marhefka <mmarhefk@redhat.com>
Date: Mon, 21 Jan 2019 09:46:38 +0100
Subject: [PATCH] Fixed bind and libreswan crypto policy test scenarios

* bind and libreswan test scenarios did not have applicability set.
  This commit adds Fedora and RHEL8 to platforms and missing OSPP
  profile to the profiles section.
---
 .../bind_not_installed.pass.sh                                 | 3 ++-
 .../libreswan_not_installed.pass.sh                            | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_bind_crypto_policy/bind_not_installed.pass.sh b/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_bind_crypto_policy/bind_not_installed.pass.sh
index a81415df04..b38329aab2 100644
--- a/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_bind_crypto_policy/bind_not_installed.pass.sh
+++ b/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_bind_crypto_policy/bind_not_installed.pass.sh
@@ -1,4 +1,5 @@
 #!/bin/bash
-# profiles = xccdf_org.ssgproject.content_profile_standard
+# platform = multi_platform_fedora, Red Hat Enterprise Linux 8
+# profiles = xccdf_org.ssgproject.content_profile_ospp, xccdf_org.ssgproject.content_profile_standard
 
 yum remove -y bind || true
diff --git a/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_libreswan_crypto_policy/libreswan_not_installed.pass.sh b/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_libreswan_crypto_policy/libreswan_not_installed.pass.sh
index 69ec02fe13..f7ecb0622d 100644
--- a/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_libreswan_crypto_policy/libreswan_not_installed.pass.sh
+++ b/tests/data/group_system/group_software/group_integrity/group_crypto/rule_configure_libreswan_crypto_policy/libreswan_not_installed.pass.sh
@@ -1,4 +1,5 @@
 #!/bin/bash
-# profiles = xccdf_org.ssgproject.content_profile_standard
+# platform = multi_platform_fedora, Red Hat Enterprise Linux 8
+# profiles = xccdf_org.ssgproject.content_profile_ospp, xccdf_org.ssgproject.content_profile_standard
 
 yum remove -y libreswan || true
